tests/providers/amazon/aws/hooks/test_eks.py

```diff
@@ -95,6 +93,12 @@
     region_matches_partition,
 )
 
+if TYPE_CHECKING:
+    from datetime import datetime
+
+    from _pytest._code import ExceptionInfo
```

Comment:
```suggestion
    from pytest import ExceptionInfo
```

Pytest now re-exports this. https://github.com/pytest-dev/pytest/commit/f2d65c85f4e5761b0e763b04932df600aec5a480